packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -21,49 +21,80 @@
       toggle="#_appSwitcherButton"
       mode="click"
       padding-size="small"
+      v-on:show-drop="updateAppIcons"
```

Comment:
For consistency across the project:

```suggestion
      @show-drop="updateAppIcons"
```